administrator/language/en-GB/joomla.ini

```
@@ -826,6 +826,13 @@ JWARNING_REMOVE_ROOT_USER_ADMIN="The emergency Root User setting is enabled for
 JWARNING_TRASH_MUST_SELECT="You must select at least one item to remove."
 JWARNING_UNPUBLISH_MUST_SELECT="You must select at least one item to unpublish."
 
+; Schemaorg
+JSCHEMAORG_EXTENSION_ALLOWED_DESCRIPTION="Activate this plugin only for listed extensions. If used all other extensions are disabled."
+JSCHEMAORG_EXTENSION_ALLOWED_LABEL="Allowed Extension"
```

```suggestion
JSCHEMAORG_EXTENSION_ALLOWED_LABEL="Allowed Extensions"
```

This is a multiselect field so it  should be a plural